docs/proposals/choosing-scheduler.md

```diff
@@ -0,0 +1,165 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest release of this document can be found
+[here](http://releases.k8s.io/release-1.1/docs/proposals/choosing-scheduler.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Choosing scheduler in a multi-scheduler configuration
+
+## Introduction
+
+This document proposes a mechanism for assigning a scheduler to a pod
+in a system which is using multiple schedulers as described [here](multiple-schedulers.md).
+In particular, we decribe an admission controller which adds the scheduler name annotation,
+and which takes its configuration from a new API object called `PodPolicy` that determines,
```

Comment:
As Tim notes later, I did not mean to say "API object" here -- this should say "type." The new API object is PodSchedulerPolicy, which corresponds to the admission controller that assigns scheduler name annotation to pod.